packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/backup.constants.js

```diff
@@ -0,0 +1,22 @@
+export const BACKUP_OFFER_LEGACY = 'legacy';
+export const BACKUP_OFFER_PREMIUM = 'premium';
+export const BACKUP_OFFER_ADVANCED = 'advanced';
+export const BACKUP_OFFER_CLASSIC = 'classic';
```

Comment:
Do we need to copy the enum values ? Maybe if needed we could generate them, `BACKUP_OFFER_${offer.toUpperCase()}`

Reply:
- An `enum` makes more sense and is easier to read imo 🙂 
- I'm becoming less and less a fan of `BACKUP_OFFER_ + something` as time goes by because I means I will never be able to `ctrl + f` all the instances of `BACKUP_OFFER_CLASSIC` for example

Reply:
Why did you resolve ? Will you do it or not ?

Reply:
@FredericEspiau I agree about readability, but we are basically copy/pasting enum from the API, meaning if an enum is added we have to add it in the constant file which means more maintenance

Reply:
Sorry, I resolved it by mistake. But we we do not have the enums defined, and if a enum is modified on the API side, we will have to change it in the entire code-base (we will have to do it only in one place if the enum is changed, for example, they rename `classic` to `veeam-classic` later on). IMO, having enums increases readability by defining certain things more explicitly